docs/support/index.md

```diff
@@ -1,6 +1,6 @@
 # Supporting materials
 
-## [Tutorials](tutorials/index.md)
```

Comment:
Should tutorials and FAQ be moved away from this index since they are no longer in the support directory?

Reply:
I think they fit on there quite well. The whole page seems to be there just in case someone types the url from memory/has a bookmark.

Reply:
#509